lua/null-ls/builtins/completion.lua

```diff
@@ -0,0 +1,34 @@
+local h = require("null-ls.helpers")
+local methods = require("null-ls.methods")
+local utils = require("null-ls.utils")
+
+local COMPLETION = methods.internal.COMPLETION
+
+local M = {}
+
+M.spell = h.make_builtin({
+    method = COMPLETION,
+    filetypes = {},
+    name = "spell",
+    runtime_condition = function(_)
+        return vim.opt_local.spell:get()
+    end,
```

Comment:
```suggestion
    generator_opts = {
        runtime_condition = function(_)
            return vim.opt_local.spell:get()
        end,
    },
```

This would need to be put inside `generator_opts` to work. Also. I'm not sure if we need to have this at all – at the moment, we don't have any built-ins with default conditions, so I think we could just mention this in the documentation as a (reasonable) suggestion.